dist/index.html

```diff
@@ -20,13 +19,13 @@ <h2>Recent scores</h2>
                         <table class="table-score-board">
     
                         </table>
-                    </div>    
+                    </div>
                 </div>
                 <div class="score-add">
                     <form class="form-add">
                         <h2>Add your score</h2>
                         <input type="text" name="name" placeholder="Your name" id="name" required>
-                        <input type="text" name="score" placeholder="Your score" id="score" required>
+                        <input type="number" name="score" placeholder="Your score" id="score" required>
                         <button type="submit" class="btn-submit">Submit</button>
                     </form>
                 </div>
```

Comment:
- [ ] The name should be a text, not a number, kindly make sure that a user doesn't send the number instead of text.

![db1](https://user-images.githubusercontent.com/55635977/151204277-14370135-7485-42c3-a6a3-4cfdf5d89879.png)

Reply:
Thank you sir for your review 👍